native_mate/native_mate/arguments.h

```diff
@@ -34,6 +35,17 @@ class Arguments {
     return ConvertFromV8(isolate_, info_->Data(), out);
   }
 
+  template <typename T>
+  bool GetNext(base::Optional<T>* out) {
+    if (next_ >= info_->Length())
+      return false;
```

Comment:
This has to be `true` @codebytere 

If there is no next but the type is optional then we should just give it an empty optional

Reply:
(and mark the `GetNext` as succeeded)

Reply:
That's the difference between this `GetNext` and the one below

Reply:
done!